backend/app/websocket.py

```diff
@@ -106,10 +106,24 @@ def process_chat_input(
             else None
         ),
         stream=True,
+        generation_config=(
+            bot.generation_config if bot and bot.generation_config else None
+        ),
     )
 
     is_anthropic = is_anthropic_model(args["model"])
+
+    args_generation_config = {
```

Comment:
Why don't we just log generated_config composed by `compose_args`?

Reply:
yes. The values in generated_config have been spread and just reconstruct here for logging